lib/perl/Genome/InstrumentData/Solexa.pm

```
@@ -764,9 +756,16 @@ sub _convert_trimmer_to_sx_commands {
 
     # Add -- to the odd, and quotes to the even, handle logicals
     my $sx_cmd = 'trim '.$trimmer_name;
-    for ( my $i = 0; $i < @params; $i += 2 ) {
+    my $i = 0;
+    while ( $i <= $#params ) {
         my $property_name = $params[$i];
+        $property_name =~ s/^\-+//;
         $property_name =~ s/\-/_/g;
+        # The far format param is not valid anymore.
+        if ( $property_name eq 'format' and $trimmer_name eq 'far' ) {
+            $i += 2;
+            next;
+        }
```

Silently skip. This is for the legacy far interface which needs a format passed to it. The format is determined in the command itself. Ideally, we could backfill processing profiles to remove the format param, and then remove it here.